lib/chef/resource/powershell_package_source.rb

```diff
@@ -24,36 +24,130 @@ class PowershellPackageSource < Chef::Resource
 
       provides :powershell_package_source
 
-      description "Use the **powershell_package_source** resource to register a PowerShell package repository."
+      description "Use the **powershell_package_source** resource to register a PowerShell Repository or other Package Source type with. There are 2 distinct objects we care about here. The first is a Package Source like a PowerShell Repository
+      or a Nuget Source. The second object is a provider that PowerShell uses to get to that source with, like PowerShellGet, Nuget, Chocolatey, etc. "
```

Comment:
This should all just go on a single line and softwraps will make it display correctly

Reply:
Cheers, done